frontend/public/components/secret.jsx

```diff
@@ -110,10 +111,18 @@ const SecretDetails = ({ obj: secret }) => {
           <div className="col-md-6">
             <ResourceSummary resource={secret} />
           </div>
+          {type && (
+            <div className="col-md-6">
+              <dl className="co-m-pane__details">
+                <dt>{t('public~Type')}</dt>
+                <dd>{type}</dd>
```

Comment:
Shouldn't this be using `DetailsItem` since it's value directly from the resource? `DetailsItem` does extra work to potentially pull out descriptions from metadata.
```suggestion
                <DetailsItem label={t('public~Type')} obj={secret} path="type" />
```

Reply:
done

Reply:
Nice!
![image](https://user-images.githubusercontent.com/14068621/114240972-6ffcae80-9956-11eb-8b6b-b4418b3f3ec6.png)